test/document_stores/test_document_store.py

```diff
@@ -453,6 +453,32 @@ def test_write_document_meta(document_store: BaseDocumentStore):
     assert document_store.get_document_by_id("4").meta["meta_field"] == "test4"
 
 
+@pytest.mark.parametrize("document_store", ["sql", "faiss", "milvus1", "milvus"], indirect=True)
```

Comment:
I wanted to test only docstores, whose metadata are written in SQL databases.
For other docstores, I think that we don't have this error...

Reply:
Oh you're right, because the other docstores would be happy to deal with lists in the meta, so they won't show this error. 

Then we might even want to test only `sql`. I got thinking here because, for example. `pinecone` uses `SQLDocumentStore` as well, and keeping track of all subclasses would be prone to errors. Let's just test `sql`, it should be enough.